### PR TITLE
irmin-http: cohttp-lwt-unix needed for tests

### PR DESCRIPTION
**Pull request by mseri**
as well as conduit-lwt-unix but that is required by cohttp-lwt-unix.

Seen on: https://github.com/ocaml/opam-repository/pull/20343 
```
Processing  1/2: [irmin-http: dune runtest]
+ /home/opam/.opam/opam-init/hooks/sandbox.sh "build" "dune" "runtest" "-p" "irmin-http" "-j" "31" (CWD=/home/opam/.opam/4.13/.opam-switch/build/irmin-http.2.6.0)
- File "test/irmin-http/dune", line 4, characters 56-72:
- 4 |  (libraries alcotest cohttp-lwt cohttp-lwt-unix conduit conduit-lwt-unix fmt
-                                                             ^^^^^^^^^^^^^^^^
- Error: Library "conduit-lwt-unix" not found.
```

Signed-off-by: Marcello Seri <marcello.seri@gmail.com>

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
`git-cohttp-unix` seems to be also needed
```
#=== ERROR while compiling irmin-http.2.6.0 ===================================#
# context              2.0.10 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/mirage/irmin/releases/download/2.6.0/irmin-2.6.0.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/irmin-http.2.6.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p irmin-http -j 31
# exit-code            1
# env-file             ~/.opam/log/irmin-http-18140-58c514.env
# output-file          ~/.opam/log/irmin-http-18140-58c514.out
### output ###
# File "test/irmin-git/dune", line 5, characters 12-27:
# 5 |    git-unix git-cohttp-unix lwt lwt.unix)
#                 ^^^^^^^^^^^^^^^
# Error: Library "git-cohttp-unix" not found.
# Hint: try:
#   dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules --default-target @install --always-show-command-line --promote-install-files --release --only-packages irmin-http -p irmin-http --profile release -j 31 @runtest
```

File: packages/irmin-http/irmin-http.2.6.0/opam [hunk 1]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.4.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.6.0/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.6.1/opam [hunk 2]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.4.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.6.1/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.7.0/opam [hunk 3]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.4.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.7.0/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.7.1/opam [hunk 4]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.4.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.7.1/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.7.2/opam [hunk 5]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.4.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.7.2/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.8.0/opam [hunk 6]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.5.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.8.0/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

File: packages/irmin-http/irmin-http.2.9.0/opam [hunk 7]
```diff
@@ -32,6 +32,7 @@ depends: [
   "irmin-test" {with-test & = version}
   "git-unix"   {with-test & >= "3.5.0"}
   "digestif"   {with-test & >= "0.9.0"}
+  "cohttp-lwt-unix" {with-test}
```
Comment (mseri) [inline packages/irmin-http/irmin-http.2.9.0/opam:35]:
```suggestion
  "cohttp-lwt-unix" {with-test}
  "git-cohttp-unix" {with-test}
```

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**